client/app/components/packages/rwcds-form/project-description.hbs

```diff
@@ -1,47 +1,39 @@
-
-  <section class="form-section">
-    <h2 class="section-header">
-      <span id="project-description" class="section-anchor"></span>
-      Project Description
-    </h2>
-
+{{#let @form as |form|}}
+  <form.Section @title="Project Description">
     <label>
       <strong>
         Has the project changed since submission of the Pre-Application Statement (PAS)?
       </strong>
       <br>
-        {{#each (array
-          (hash
-            code=(optionset 'rwcdsForm' 'dcpHasprojectchangedsincesubmissionofthepas' 'code' 'YES')
-            label=(optionset 'rwcdsForm' 'dcpHasprojectchangedsincesubmissionofthepas' 'label' 'YES'))
-          (hash
-            code=(optionset 'rwcdsForm' 'dcpHasprojectchangedsincesubmissionofthepas' 'code' 'NO')
-            label=(optionset 'rwcdsForm' 'dcpHasprojectchangedsincesubmissionofthepas' 'label' 'NO'))
-        ) as |radio|}}
-          <@rwcdsForm.radio
-            @value={{radio.code}}
-            @groupValue={{@rwcdsForm.saveableChanges.dcpHasprojectchangedsincesubmissionofthepas}}
-            data-test-radio="dcpHasprojectchangedsincesubmissionofthepas"
-            data-test-radio-option={{radio.label}}
-          >
-            {{radio.label}}
-          </@rwcdsForm.radio>
-        {{/each}}
+      <form.Field
+        @type="radio-group"
+        @attribute="dcpHasprojectchangedsincesubmissionofthepas"
+      as |RadioGroup|>
```

Comment:
I don't feel super strong about this but in other parts of the app we do the spacing differently:
```handlebars
        <form.Field
          @type="radio-group"
          @attribute="dcpHasprojectchangedsincesubmissionofthepas"   as |RadioGroup|
>
```

I think your way makes more sense.